audit-logger/package.json

```
@@ -35,7 +35,7 @@
   },
   "private": true,
   "engines": {
-    "node": "18"
+    "node": "16"
```

Pour avoir la même version que pour les autres applications

```suggestion
    "node": "^16.20.1"
```